src/status.c

```diff
@@ -738,6 +746,8 @@ status_request(struct view *view, enum request request, struct line *line)
 		return request;
 	}
 
+	if (view->parent)
```

Comment:
These 2 lines are problematic. When present, if you open the Untracked only status view from the Untracked files line, stage one file, open the regular status view (with s key) and then stage another file then the refresh is not correct anymore. @jonas, any idea how to achieve the automatic refresh of the parent view properly ?

Reply:
Trigger a reload of the main view? Some views like stage has explicit logic based on the parent view if I recall correctly.

Reply:
I noticed it and I also noticed there is no direct call to update_view, but I could not spot which part is the trigger for this logic.

Reply:
I found it ! update_view is actually called for parent in main_status_exists itself called by stage_exists at the end of stage_request. So my code should be correct.